### PR TITLE
Initialise Stability Inspector

### PR DESCRIPTION
**Pull request by wilson1000**
Adds the base files

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/variables.tf [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  default     = "Justice Gov UK Archive"
+}
+
+variable "namespace" {
+  description = "Name of the namespace these resources are part of"
+  type        = string
+  default     = "stability-inspector-dev"
+}
+
+variable "archive_date" {
```
Comment (EarthlingDavey) [inline namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/variables.tf:23]: Probably left over from the archiver repo?

File: namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/variables.tf [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+variable "oidc_name" {
+  description = "Name of the OIDC provider (github or circleci)"
+  type        = string
+  default     = "github"
+}
+
+variable "app_repo" {
+  description = "Name of application repository"
+  type        = string
+  default     = "justice-website-archive"
```
Comment (EarthlingDavey) [inline namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/variables.tf:86]: `justice-website-archive` :)

**Review by EarthlingDavey — CHANGES_REQUESTED** [review]

File: namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/push_gateway.tf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module "pushgateway" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-pushgateway?ref=version" # use the latest relase
```
Comment (EarthlingDavey) [inline namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/push_gateway.tf:2]: Nice find with this:) It looks like `version` needs to be `1.4.0`

File: namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/variables.tf (hunk 1 above)
Comment (wilson1000) [reply namespaces/live.cloud-platform.service.justice.gov.uk/stability-inspector-dev/resources/variables.tf]: Thank you :)

**Comment by EarthlingDavey** [comment]
Looks good to me!

**Review by wilson1000 — COMMENTED** [review]
Changes have been reviewed.

**Review by timckt — DISMISSED** [review]

**Comment by EarthlingDavey** [comment]
@wilson1000 Changes look good. I missed a few instances of 'archive' in the files on my review 😅

If you go to the files changed tab, it's mentioned 3 times.

**Review by EarthlingDavey — APPROVED** [review]

**Review by timckt — APPROVED** [review]

**Pull request merged**